package.json

```diff
@@ -25,10 +25,9 @@
     "homebridge",
     "smartthings-routines"
   ],
-  "author": "Alexander J. V. White <hello@ajvwhite.com> (http://ajvwhite.com)",
   "license": "MIT",
   "bugs": {
-    "url": "https://github.com/ajvwhite/homebridge-smartthings-routine-triggers/issues"
+    "url": "https://github.com/needlerp/homebridge-smartthings-routines/issues"
```

Comment:
For a pull request - homepage references and URLs need to remain as it - of course we should add you as a contributor to the readme, and your input is much appreciated